File: 012.jarakirefactor/test/ebin/sequencia5.erl
-module(sequencia5).

-compile(export_all).

-import(loop, [for/3, while/2]).

main(V_args) ->
    st:new(),
    st:get_new_stack(main),
    st:put_value({main, "args"}, {'String', V_args}),
    io:format("~s~n",
	      ["Informe o numero em metros ser convertido"]),
    st:put_value({main, "metro"},
		 {int, io:fread(">", "~d")}),
    st:put_value({main, "centimetro"},
		 {int, st:get_value(main, "metro") * 100}),
    io:format("~p~s~p~s~n",
	      [st:get_value(main, "metro"), " metros corresponde a ",
	       st:get_value(main, "centimetro"), " centimetros  "]),
    st:get_old_stack(main),
    st:destroy().

File: java_src/Sequencia5.java
package sequencia;

import java.util.Scanner;

public class Sequencia5 {
	public static void main(String[] args){

		int metro, centimetro;
		
		Scanner input = new Scanner(System.in);
		
		System.out.println("Informe o numero em metros ser convertido");
		metro = input.nextInt();

		centimetro = metro*100;	
		
		System.out.println(metro + " metros corresponde a " + centimetro + " centimetros  ");
	}
}
